### ISSUE TITLE
Share downloads folder across instances of the same game.

### ISSUE THREAD

**Issue by ProgrammerFailure** [ModOrganizer2/modorganizer#2077]
**Is your feature request related to a problem? Please describe.**
I use MO2 for Skyrim Special edition. When I need different FOMOD options etc, I create a new instance. However, each instance has its own download folder, so I need to duplicate lots of files, using tons of extra space.

Second issue which would be fixed with the proposed solution: installing a mod in a profile installs it to every profile in the same instance.

**Describe the solution you'd like**
Downloads and metadata folders should be separated, with 1 downloads folder per managed game (not instance) and a metadata folder for each profile. This allows profile/instance specific installation while minimizing duplicate files (only some metadata and mods identical across games are duplicated).

**Describe alternatives you've considered**
An alternative is making profile specific metadata but keeping instance-specific downloads folder. Then, users can use symlinks to share downloads folders between instances. 

This doesn't work currently because metadata is stored in the downloads folder. Hence, if I symlink two instances to the same downloads folder, mods installed in instance A show up as installed in instance B.

See: second issue which would be fixed with the proposed solution.

**Additional context**
None

**Comment by ProgrammerFailure** [comment]
Actually, I completely forgot about just changing the downloads folder paths. The issues with metadata being the same across profiles/instances remain though.

**Comment by UhuruNUru** [comment]
Old school MO Tip, for FoMods if you select different options in different profiles, is manually install the FoMod.
In most cases you simply put each option you might choose in a different "Mod" folder in MO2.
A suitable naming structure is most often found by simply using the folder paths the Mod Author used, fo0r the same reasns, you just convert the path into a mod name, replacing slashes "\" with spaced hyphen " - " which makes it easier to read.

Clearly if you never pick other options in a FoMod this is a waste of effort, but for mods with many options, like Body replacers, that you may change per profile, it's a far better method than profile specific.

The biggest advantage is you can tell what options you selected at a glance, even years later when you have forgotten what you selected.

**Comment by ProgrammerFailure** [comment]
> Old school MO Tip, for FoMods if you select different options in different profiles, is manually install the FoMod.
> In most cases you simply put each option you might choose in a different "Mod" folder in MO2.
> A suitable naming structure is most often found by simply using the folder paths the Mod Author used, fo0r the same reasns, you just convert the path into a mod name, replacing slashes "\" with spaced hyphen " - " which makes it easier to read.
> 
> Clearly if you never pick other options in a FoMod this is a waste of effort, but for mods with many options, like Body replacers, that you may change per profile, it's a far better method than profile specific.
> 
> The biggest advantage is you can tell what options you selected at a glance, even years later when you have forgotten what you selected.

This works, but still it would be better for  this to be baked in to MO2. I will start using it though. And the other thing about different instances and wasted space is still true.

**Comment by UhuruNUru** [comment]
MO pro tip use active filter, if you only want to see the mods for that specific Profile.
The filters are there for a reason, I've had 1,000s of mods installed on many different profile in the past.

When you want to add a mod, take it off to add ones you already have on another profile.
Remember some FoMod changes may need to be reapplied after installing new mods.
You can manually split FoMods with many options, make each option a sub mod, noting exclusive Alternates (pick one) & mult-choice options (tick boxes). It's your Profile, you decide what each MO2 Mod contains.

The real power of MO is that it is not messed up with your manual changes outside MO & by being modular you can control ANY mods parts independently by simply making that part a separate sub-mod of the core mod folder in MO.

Basically with no filters applied you see every mod on every profile by design, that's exactly what you need to see when adding mods you've already installed on other profiles, but otherwise stick to the active filter to see the just the mods being used.

**Comment by Quest79** [comment]
Honestly the downloads folder should be separated from the profile settings. You should still be able to change it but it should be coded in as its own tab for managing, given why the hell would we have a separate downloads folder 99 cases out of 100?

**Comment by UpstairsEmpire** [comment]
@Quest79 The downloads folder is separated from profiles isn't it?
